### PR TITLE
Sync code of conduct

### PR DESCRIPTION
**Pull request by ErikSchierboom**
@iHiD I've taken this from https://github.com/exercism/.github/blob/main/CODE_OF_CONDUCT.md

### CODE REVIEW

File: synced-files/CODE_OF_CONDUCT.md
Comment (iHiD) [inline synced-files/CODE_OF_CONDUCT.md:4]: We can scrap this now

```suggestion
```

**Comment by iHiD** [comment]
This is the "official" one: https://github.com/exercism/website-copy/blob/main/pages/code_of_conduct.md

It's probably identical to the one you used though.

File: synced-files/CODE_OF_CONDUCT.md
Comment (ErikSchierboom) [reply synced-files/CODE_OF_CONDUCT.md]: I updated to the https://github.com/exercism/website-copy/blob/main/pages/code_of_conduct.md version.

File: global-files/CODE_OF_CONDUCT.md
Comment (SaschaMann) [inline global-files/CODE_OF_CONDUCT.md:82]:
```suggestion
_This policy is a "living" document, and subject to refinement and expansion in the future. This policy applies to the Exercism website, the Exercism GitHub organisation, any other Exercism-related communication channels (e.g. Slack, Twitter, Email) and any other Exercism entity or event._
```

Comment (ErikSchierboom) [reply global-files/CODE_OF_CONDUCT.md:82]: Why use a capital E for email?

```suggestion
_This policy is a "living" document, and subject to refinement and expansion in the future. This policy applies to the Exercism website, the Exercism GitHub organisation, any other Exercism-related communication channels (e.g. Slack, Twitter, email) and any other Exercism entity or event._
```

File: .github/CODEOWNERS
Comment (SaschaMann) [inline .github/CODEOWNERS:6]: Why are there leading `/` but not above?

File: .github/CODEOWNERS
Comment (SaschaMann) [inline .github/CODEOWNERS:3]:
```suggestion

# Config
config.toml @exercism/maintainers-admin

```

This determines which files are appendable and should also be protected.

**Review by SaschaMann — COMMENTED** [review]
It's worth noting that anyone with write access to this repo indirectly gets write access to all repos due to exercism-bot. As a result, the codeowners don't really make a difference as one could bypass them. There's no harm in having them but it's somewhat redundant if you heavily restrict write access to the repo anyway.

If that's an issue, the bot token could be moved to a github environment and require manual sign-off by the admins. However, that would make testing changes in this repo an absolute nuisance for anyone but Erik :D

File: global-files/CODE_OF_CONDUCT.md
Comment (iHiD) [reply global-files/CODE_OF_CONDUCT.md]: comma is good, but it should be 'email' (as it's a "common noun" in english now)

File: .github/CODEOWNERS
Comment (ErikSchierboom) [reply .github/CODEOWNERS:6]: Because this is a directory and the above is a file. At least that is how I interpreted it in: https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-code-owners#codeowners-syntax

Comment (SaschaMann) [reply .github/CODEOWNERS:9]: In v3 we used `reference/stories/**/*` for dirs fwiw.

**Review by SaschaMann — APPROVED** [review]

File: global-files/CODE_OF_CONDUCT.md
Comment (petertseng) [inline global-files/CODE_OF_CONDUCT.md:15]: accidently should be accidentally

Comment (SaschaMann) [reply global-files/CODE_OF_CONDUCT.md:15]: #58 fixes it. It's not worth sending out a bulk PR for that alone but it'll be synced once people update an append or the next big PR happens.

Comment (ErikSchierboom) [reply global-files/CODE_OF_CONDUCT.md:15]: Lol!

**Pull request merged**
